src/sentry/static/sentry/app/views/organizationEventsV2/transactionView/utils.tsx

```diff
@@ -0,0 +1,272 @@
+import {isString} from 'lodash';
+
+type Rect = {
+  // x and y are left/top coords respectively
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+};
+
+// get position of element relative to top/left of document
+const getOffsetOfElement = (element: Element) => {
+  // left and top are relative to viewport
+  const {left, top} = element.getBoundingClientRect();
+
+  // get values that the document is currently scrolled by
+  const scrollLeft = window.pageXOffset || document.documentElement.scrollLeft;
+  const scrollTop = window.pageYOffset || document.documentElement.scrollTop;
```

Comment:
The [pageYOffset attribute](https://developer.mozilla.org/en-US/docs/Web/API/Window/pageYOffset) has pretty solid support. Do we need a fallback?